test-cases/longevity/longevity-schema-changes-3h.yaml

```diff
@@ -11,7 +11,8 @@ n_loaders: 2
 n_monitor_nodes: 1
 round_robin: true
 
-instance_type_db: 'i3en.large'
+hinted_handoff: 'disabled'  # A workaround for issues like: https://github.com/scylladb/scylla-enterprise/issues/3102
+instance_type_db: 'i4i.2xlarge'
```

Comment:
This is not necessarily needed.
We didn't see a run with "hinted_handoff" and i3en.large.

Reply:
Tested in https://jenkins.scylladb.com/job/scylla-staging/job/yarongilor/job/longevity-schema-changes-3h-test-repro-3102/8/

Reply:
@roydahan , @fruch , i accidentally reran the same test (hinted_handoff: disabled instance_type_db: i4i.2xlarge) and c-s failed with a read timeout:
https://jenkins.scylladb.com/job/scylla-staging/job/yarongilor/job/longevity-schema-changes-3h-test-repro-3102/8/

```
2023-07-06 11:48:44.121: (CassandraStressLogEvent Severity.CRITICAL) period_type=one-time event_id=2de061ed-fb7b-4c1a-850b-3c62a26d74f6 during_nemesis=AddRemoveMv: type=OperationOnKey regex=Operation x10 on key\(s\) \[ line_number=1076 node=Node longevity-schemachanges-3h-repro-31-loader-node-7a0532d6-1 [3.238.144.154 | 10.12.0.221] (seed: False)
java.io.IOException: Operation x10 on key(s) [50373830304f344b3031]: Error executing: (ReadTimeoutException): Cassandra timeout during read query at consistency QUORUM (2 responses were required but only 1 replica responded). In case this was generated during read repair, the consistency level is not representative of the actual consistency.
```

Reply:
A similar run with `i3en.large` failed 5 minutes after one node is stopped, for read-timeout as well
[argus](https://argus.scylladb.com/test/2c336432-92be-4a04-88d7-5e31fb79a606/runs?additionalRuns[]=7a48057c-766b-4dda-8488-c2b00ad2cdc5)

Reply:
So, now we need to "decide" if this load is too heavy for this cluster or it's a scylla issue.
This is the hardest part for any new test....

I don't know what's the best way to do it.
You can start by consulting someone about this "timeout" (with the full semaphore timeout from the log) and then decide how to proceed (lower load / bigger cluster).

Reply:
Start by showing what's the throughput you have and what's the disk writes/reads bandwidth